### PR TITLE
ncm-metaconfig: basic sysconfig for ceph

### PR DESCRIPTION
**Pull request by kwaegema**

### CODE REVIEW

File: ncm-metaconfig/src/main/metaconfig/ceph/sysconfig.tt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[%- FOR pair IN CCM.contents -%]
```
Comment (stdweird) [inline ncm-metaconfig/src/main/metaconfig/ceph/sysconfig.tt]: `CCM.contents.pairs`

File: ncm-metaconfig/src/main/metaconfig/ceph/tests/regexps/config [hunk 2]
```diff
@@ -0,0 +1,7 @@
+Base test for config
+---
+/etc/sysconfig/ceph
+unordered
```
Comment (stdweird) [inline ncm-metaconfig/src/main/metaconfig/ceph/tests/regexps/config]: should be ordered by `.pairs`, so remove this

File: ncm-metaconfig/src/main/metaconfig/ceph/pan/schema.pan [hunk 3]
```diff
@@ -0,0 +1,11 @@
+declaration template metaconfig/ceph/schema;
+
+include 'pan/types';
+
+
+@{ type for configuring the ceph sysconfig file @}
```
Comment (stdweird) [inline ncm-metaconfig/src/main/metaconfig/ceph/pan/schema.pan]: remove final `@`

File: ncm-metaconfig/src/main/metaconfig/ceph/pan/schema.pan [hunk 4]
```diff
@@ -0,0 +1,11 @@
+declaration template metaconfig/ceph/schema;
+
+include 'pan/types';
+
+
```
Comment (stdweird) [inline ncm-metaconfig/src/main/metaconfig/ceph/pan/schema.pan]: remove newline

**Comment by kwaegema** [comment]
@stdweird comments processed

**Comment by stdweird** [comment]
LGTM

**Pull request merged**